GNUmakefile

```
@@ -408,3 +413,16 @@ ec2info: ## Generate AWS EC2 CPU specification table
 .PHONY: cl
 cl: ## Create a new Changelog entry
 	@go run -modfile tools/go.mod tools/cl-entry/main.go
+
+# Use this target as a smoke test
+.PHONY: test
+test: GOTEST_PKGS := $(foreach g,$(GOTEST_GROUP),$(shell go run -modfile=tools/go.mod tools/missing/main.go ci/test-core.json $(g)))
+test:
```

```suggestion
.PHONY: test
test: GOTEST_PKGS := $(foreach g,$(GOTEST_GROUP),$(shell go run -modfile=tools/go.mod tools/missing/main.go ci/test-core.json $(g)))
test: ## Use this target as a smoke test
```

This convention for "docstrings" lets the target get picked up by the silly little awk script we have in `make help`